backend/routes/books.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const controller = require('../controllers/books');
```

Comment:
Separate library and project-related imports by new line. Also group this kinds of imports together